dashboard/test/models/teacher_feedback_test.rb

```diff
@@ -179,4 +179,46 @@ class TeacherFeedbackTest < ActiveSupport::TestCase
       assert_equal datetime2, feedback.student_last_visited_at
     end
   end
+
+  test 'get_script_level finds level in script' do
+    script_level = create :script_level
+    script = script_level.script
+    level = script_level.levels.first
+    feedback = create :teacher_feedback, script: script, level: level
+    assert_queries(1) do
+      assert_equal script_level, feedback.get_script_level
+    end
+  end
+
+  test 'get_script_level finds bubble choice parent level' do
+    parent_level = create :bubble_choice_level, :with_sublevels
+    child_level = parent_level.sublevels.first
+
+    # Create these intermediate rungs of the hierarchy, so that script_level
+    # will show up in script.script_levels.
+    script = create :script
+    lesson_group = create :lesson_group, script: script
+    lesson = create :lesson, lesson_group: lesson_group, script: script
```

Comment:
No need to address as part of this PR, but I'm curious if there's anything we can do in factories to short cut knowing and creating each rung?

Reply:
yes - when creating a script level, the rung that is missing is lesson group. there should be a way to make the lesson factory create a lesson group, the same way the script level factory creates a lesson: https://github.com/code-dot-org/code-dot-org/blob/62b10f0b330f124cff39744c43863cf4e0e4ff86/dashboard/test/factories/factories.rb#L725-L727

I tried this, but it didn't work using similar syntax.